packages/breadboard-web/src/boards/open-alex-entity-search-list.ts

```diff
@@ -4,177 +4,108 @@
+  entity: entity,
+  page: page,
+  per_page: per_page,
+  search: search,
+  select: select,
+});
 
-  const response = spread({ $id: "spreadResponse", object: fetchUrl.response });
+const fetcher = fetch({
+  $id: "fetch",
+  method: "GET",
+  url: urlTemplater.outputs.url,
+});
 
-  const output = base.output({
-    schema: {
-      type: "object",
-      properties: {
-        url: {
-          type: "string",
-          description: "URL to fetch",
-          title: "URL",
-        },
-        meta: {
-          type: "object",
-          properties: {
-            count: {
-              type: "integer",
-            },
-            db_response_time_ms: {
-              type: "integer",
-            },
-            page: {
-              type: "integer",
-            },
-            per_page: {
-              type: "integer",
-            },
-            groups_count: {
-              type: ["integer", "null"],
-            },
-          },
-          required: [
-            "count",
-            "db_response_time_ms",
-            "page",
-            "per_page",
-            "groups_count",
-          ],
-        },
-        results: {
-          type: "array",
-          items: {
-            type: "object",
-            properties: {
-              id: {
-                type: "string",
-              },
-              display_name: {
-                type: "string",
-              },
-              title: {
-                type: "string",
-              },
-              relevance_score: {
-                type: "number",
-              },
-            },
-            additionalProperties: true,
-          },
-        },
-        group_by: {
-          type: "array",
-          items: {},
-        },
-      },
-      required: ["url", "meta", "results", "group_by"],
-      additionalProperties: false,
+const { spread } = code(
+  {
+    $id: "spreadResponse",
+    $metadata: {
+      title: "Spread",
+      description: "Spread the properties of an object into a new object",
     },
-    $id: "response",
-    url: urlTemplate.url,
-    meta: response.meta,
-    results: response.results,
-    group_by: response.group_by,
-  });
-  return output;
+    obj: fetcher.outputs.response
+  },
+  { spread: object({}, "string") },
+  ({ obj }) => {
+    if (typeof obj !== "object") {
+      throw new Error(`object is of type ${typeof obj} not object`);
+    }
+    const spread = { ...obj }
+    return { spread };
+  }
+).outputs;
+
+const results = output(spread, {
+  title: "Entity Search Results",
+  description: "A list of entities from the search results ",
 });
 
-export default await graph.serialize({
+export default board({
   title: "Open Alex Entity Search Results",
   description: "Query the OpenAlex API for a list entities",
   version: "0.0.1",
+  inputs: {
+    entity,
+    page,
+    per_page,
+    search,
+    select
+  },
+  outputs: { results }
```

Comment:
The previous version also says it returns `url`, `meta`, and `group_by`. I'm not sure if those are useful? If they matter I guess we should include them. If not, sounds good to ignore them.

Reply:
Following discussions with Joe, I've re-added back in `url` and `meta`, however `group_by` has been removed for now since it can't be used when selecting fields. Will be taking another look at it tomorrow though!

Reply:
Still not quite done with this one, so will be taking another look tomorrow!

Reply:
I've discussed this with Joe and we will open a new issue to create a separate Open Alex entity search board capable of using group by.